bin/reth/src/commands/import_op.rs

```diff
@@ -124,8 +124,8 @@ impl ImportOpCommand {
             total_decoded_txns += file_client.bodies_len();
 
             for (block_number, body) in file_client.bodies_iter_mut() {
-                body.transactions.retain(|tx| {
-                    if is_duplicate(tx.hash, *block_number) {
```

Comment:
hmm, are the dup transactions the only transactions in their blocks? Or do the headers with duplicate transactions always contain an empty transactions root? it would seem like the previous check is more correct

Reply:
yeah they are the only transactions in their blocks, I verified. agreed that checking tx hash feels more intuitive, but then again, this is a specific solution to a specific problem, so loss of generality is fine for less LOC.